src/System.Drawing.Common/tests/BufferedGraphicsContextTests.cs

```diff
@@ -254,7 +254,9 @@ public void Finalize_Invoke_Success()
         {
             // Don't allocate anything as this would leak memory.
             // This makes sure than finalization doesn't cause any errors or debug assertions.
-            var context = new BufferedGraphicsContext();
+            using (var context = new BufferedGraphicsContext())
+            {
+            }
```

Comment:
This no longer matches up with the comment:
```
This makes sure than finalization doesn't cause any errors or debug assertions.
```

Reply:
Ah, you're right. Although the test "as is" wasn't guaranteed to invoke the finalizer, unless that was a feature of xunit I was unaware of.

I updated the test to add `context = null`, call `GC.Collect` and wait for the finalizers to run. Hope that is fine.